.circleci/config.yml

```diff
@@ -0,0 +1,54 @@
+version: 2
+jobs:
+  tests:
+    docker:
+      # Primary docker container where the tests will be run
+      - image: ubuntu:18.04
```

Comment:
I wonder if we should install the miniconda docker instead?
https://hub.docker.com/r/continuumio/miniconda3

Looks like this is based on `debian`
https://github.com/ContinuumIO/docker-images/blob/54e32409288d2b0c6ea61d5027d3790f04695eef/miniconda3/Dockerfile